integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,10 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// TestRunWithoutMemoryswapLimit should run without memory swap limit.
```

Comment:
Hi @runcom , you mean that we should remove the previous comment and don't add new comment, right?